server/server.go

```diff
@@ -258,7 +258,7 @@ func (s *Server) loadSandbox(id string) error {
 		return err
 	}
 	if err = s.runtime.UpdateStatus(scontainer); err != nil {
-		return fmt.Errorf("error updating status for pod sandbox infra container %s: %v", scontainer.ID(), err)
+		logrus.Warnf("error updating status for pod sandbox infra container %s: %v", scontainer.ID(), err)
```

Comment:
Nope, on a reboot, this will keep the status as nil forcing it to be returned to kube as UNKNOWN (in sandbox_list) and kube recreating the sandbox.